app/assets/stylesheets/admins/article/_index_table.scss

```diff
@@ -0,0 +1,207 @@
+.wrapper-table {
+  display: table;
+  table-layout: fixed;
+  width: 100%;
+  margin: 0 0 40px 0;
+  font-size: 16px;
+}
+
+.header-table {
+  display: table-row;
+  height: 60px;
+  background: $background-color-title;
+  font-weight: 400;
+  color: $white-color;
+
+  .header-cell:first-child {
+    width: 29%;
+    padding: 6px 12px 6px 30px;
+    border-top-left-radius: 30px; 
+    border-bottom-left-radius: 30px;
+  }
+
+  .header-cell:nth-child(2) {
+    width: 18%;
+  }
+
+  .header-cell:nth-child(3) {
+    width: 10%;
+  }
+
+  .header-cell:nth-child(4) {
+    width: 8%;
+  }
+
+  .header-cell:nth-child(5) {
+    width: 10%;
+  }
+
+  .header-cell:last-child {
+    width: 24%;
+    padding: 6px 30px 6px 12px;
+    border-bottom-right-radius: 30px; 
+    border-top-right-radius: 30px; 
+  }
+
+  .header-cell {
+    display: table-cell;
+    vertical-align: middle;
+    padding: 6px 12px 6px 12px;
```

Comment:
change to padding: 6px 12px; Reading documentation

Reply:
corrected